### PR TITLE
Use serializable config & add multi folder support

### PR DESCRIPTION
**Pull request by 401U**
1. Use the new Serializable config.
2. Add multi folder support
3. Fix typo error in README

In 1.17+, the entities data was moved out of region file, so the region update won't sync the entities, this PR added support for multi-folder config in `dimension_region_folder` so everything works fine now.

### CODE REVIEW

File: mcdreforged.plugin.json [hunk 1]
```diff
@@ -1,12 +1,15 @@
 {
 	"id": "region_file_updater",
-	"version": "1.4.1",
+	"version": "1.4.2",
```
Comment (Fallen-Breath) [inline mcdreforged.plugin.json]: Minor version bump suggested

File: README.md [hunk 2]
```diff
@@ -40,5 +40,28 @@ A MCDR plugin to help you update region files in game
 
 # 例子 / Sample
 
-`!!region add 0 3 2` 添加主世界的r.3.2.mca至更新列表 / add overworld's r.3.2.mca to the updating list
+`!!region add 0 3 2` 添加主世界的r.3.2.mca至更新列表 / add overworld 's r.3.2.mca to the updating list
```
Comment (Fallen-Breath) [inline README.md]: what's the extra space character in `overworld 's` for

Comment (401U) [reply README.md]: Just a mistake, fixed

File: mcdreforged.plugin.json (hunk 1 above)
Comment (401U) [reply mcdreforged.plugin.json]: Done

**Comment by 401U** [comment]
Conversations resolved, able to merge now

File: mcdreforged.plugin.json [hunk 3]
```diff
@@ -1,17 +1,20 @@
...
+	"version": "1.5.0",
 	"name": "Region file Updater",
 	"description": {
 		"en_us": "A MCDR plugin to help you update region files in game",
 		"zh_cn": "一个从指定位置拉取region文件至本服存档的插件"
 	},
-	"author": "Fallen_Breath",
+	"author": [
+		"Fallen_Breath",
+		"YehowahLiu"
```
Comment (Fallen-Breath) [inline mcdreforged.plugin.json]: You are just a feature contributor, not an author

Comment (401U) [reply mcdreforged.plugin.json]: No aware about that before, should be resolved now

File: region_file_updater/__init__.py [hunk 4]
```diff
@@ -1,27 +1,26 @@
...
 PLUGIN_METADATA = ServerInterface.get_instance().as_plugin_server_interface().get_self_metadata()
-config = {
-	'enabled': True,
-	'source_world_directory': './qb_multi/slot1/world',
-	'destination_world_directory':  './server/world',
-	'dimension_region_folder':  {
+
+
+class Config(Serializable):
+	enabled: bool = True,
```
Comment (Fallen-Breath) [inline region_file_updater/__init__.py:13]: SyntaxError due to the `,` here

**Pull request merged**